app/models/taxonomy.rb

```diff
@@ -218,6 +218,17 @@ def notification_recipients_ids
     self.subtree.flat_map(&:users).map(&:id).uniq
   end
 
+  # note - this method used by before_destroy callbacks in extension files
```

Comment:
extensions files in plugins I guess, from this comment it's not clear that it should not be removed as a dead code